sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py

```diff
@@ -43,13 +46,16 @@ def __init__(
         identity: Optional[IdentityConfiguration] = None,
         primary_user_assigned_identity: Optional[str] = None,
         managed_network: Optional[ManagedNetwork] = None,
+        feature_store_settings: Optional[FeatureStoreSettings] = None,
         **kwargs,
     ):
 
         """Azure ML workspace.
 
         :param name: Name of the workspace.
         :type name: str
+        :param kind: Kind of the workspace. Default value is 'Default'
```

Comment:
The SDK interface should be lowercase strings or case insensitive. Also what're possible values other than "default"?

Reply:
from the client, there is no value mentioned as a list, right now, we have "Default" and "FeatureStore".
it can be "default" or "featurestore"

Reply:
from the response in the api, we get the kind as "Default"